book/src/crates.md

```diff
@@ -0,0 +1,44 @@
+# Create breakdown
```

Comment:
```suggestion
# Crate breakdown
```

Reply:
whoops, yeah, I shouldn't have added this